test/4_permissions_test.js

```diff
@@ -181,52 +200,40 @@ contract('LinniaPermissions', accounts => {
   });
   describe('grant policy based access', () => {
     before('set up a permissionPolicy mock contract', async () => {
-      permissionPolicyInstance = await permissionPolicy.new({from: admin});
-      permissionPolicyContractAddress = permissionPolicyInstance.address;
+      policyInstance = await Policy.new({from: admin});
+      policyContractAddress = policyInstance.address;
     });
     it('should allow user to grant policy based access to their data', async () => {
       const fakeIpfsHash = eutil.bufferToHex(crypto.randomBytes(32));
       const tx = await instance.grantPolicyBasedAccess(
         testDataHash1,
         provider2,
         fakeIpfsHash,
-        [permissionPolicyContractAddress],
+        [policyContractAddress],
         { from: user1 }
       );
 
-      const expectedArgs =  {
-        'dataHash': testDataHash1,
-        'dataUri': fakeIpfsHash,
-        'viewer': provider2,
-        'policy': permissionPolicyContractAddress,
-        'isOk': true,
-        'sender': user1
-      };
-
-      assert.equal(tx.logs[0].event, 'LinniaPolicyChecked');
```

Comment:
It would be good to check for the new event logs.

Reply:
The new event logs aren't available in the returned transaction logs. I test for them in the method that emits them though